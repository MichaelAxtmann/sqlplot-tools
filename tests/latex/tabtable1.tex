line1
% IMPORT-DATA test test.data
line2
%% TABTABLE SELECT testsize AS x, FLOOR(bandwidth), FLOOR(rate * 1e12), areasize
%% FROM test ORDER BY x, bandwidth LIMIT 10
line3
indentation start
  %% TABTABLE SELECT testsize AS x, FLOOR(bandwidth), FLOOR(rate * 1e12), areasize
  %% FROM test ORDER BY x, bandwidth
         1152	20004857425	 399	       1024
         1152	21256634665	 376	       1024
         2176	20059162540	 398	       2048
         2176	21206797610	 377	       2048
         3200	20101648154	 397	       3072
         3200	21256759869	 376	       3072
         4224	20078735758	 398	       4096
  % END TABTABLE SELECT testsize AS x, bandwidth, rate, areasize FROM test ORDER BY x

  %% TABTABLE SELECT testsize AS x, FLOOR(bandwidth), FLOOR(rate * 1e12), areasize
  %% FROM test ORDER BY x, bandwidth LIMIT 20
         1152	20004857425	 399	       1024
         1152	21256634665	 376	       1024
         2176	20059162540	 398	       2048
         2176	21206797610	 377	       2048
         3200	20101648154	 397	       3072
         3200	21256759869	 376	       3072
         4224	20078735758	 398	       4096
  % END TABTABLE SELECT testsize AS x, bandwidth, rate, areasize FROM test ORDER BY x

indentation end
this is the end