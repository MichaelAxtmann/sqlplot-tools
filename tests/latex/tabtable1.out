line1
% IMPORT-DATA test test.data
line2
%% TABTABLE SELECT testsize AS x, FLOOR(bandwidth), FLOOR(rate * 1e12), areasize
%% FROM test ORDER BY x, bandwidth LIMIT 10
1152	20004857425	399	1024
1152	21256634665	376	1024
2176	20059162540	398	2048
2176	21206797610	377	2048
3200	20101648154	397	3072
3200	21256759869	376	3072
4224	20078735758	398	4096
4224	21256469275	376	4096
6272	20103335011	397	6144
6272	21254940419	376	6144
% END TABTABLE SELECT testsize AS x, FLOOR(bandwidth), FLOOR(rate * 1e12), ar...
line3
indentation start
  %% TABTABLE SELECT testsize AS x, FLOOR(bandwidth), FLOOR(rate * 1e12), areasize
  %% FROM test ORDER BY x, bandwidth
         1152	20004857425	 399	       1024
         1152	21256634665	 376	       1024
         2176	20059162540	 398	       2048
         2176	21206797610	 377	       2048
         3200	20101648154	 397	       3072
         3200	21256759869	 376	       3072
         4224	20078735758	 398	       4096
         4224	21256469275	 376	       4096
         6272	20103335011	 397	       6144
         6272	21254940419	 376	       6144
         8320	19844289277	 403	       8192
         8320	21033549756	 380	       8192
        12416	19934071922	 401	      12288
        12416	21105216385	 379	      12288
        16512	19970353548	 400	      16384
        16512	21142325436	 378	      16384
        20608	19997700036	 400	      20480
        20608	21164943036	 377	      20480
        24704	20017094792	 399	      24576
        24704	21177968064	 377	      24576
        28800	20024361016	 399	      28672
        28800	21188091979	 377	      28672
        32896	19588722596	 408	      32768
        32896	20167090530	 396	      32768
        41088	13352542574	 599	      40960
        41088	15434595485	 518	      40960
        49280	13352656249	 599	      49152
        49280	15445744156	 517	      49152
        65664	13353218261	 599	      65536
        65664	15470586592	 517	      65536
        98432	13353616903	 599	      98304
        98432	15476290580	 516	      98304
       131200	13353488133	 599	     131072
       131200	15478200720	 516	     131072
       196736	13353703915	 599	     196608
       196736	15452726013	 517	     196608
       262272	13353783611	 599	     262144
       262272	15403930682	 519	     262144
       393344	13354375997	 599	     393216
       393344	15424185515	 518	     393216
       524416	13355357557	 599	     524288
       524416	15423989946	 518	     524288
       786560	13357098956	 598	     786432
       786560	15426535975	 518	     786432
      1048704	13322300787	 600	    1048576
      1048704	15416610336	 518	    1048576
      1310848	12599911132	 634	    1310720
      1310848	15270216963	 523	    1310720
      1572992	12597385271	 635	    1572864
      1572992	15269613288	 523	    1572864
      1835136	12593497036	 635	    1835008
      1835136	15269091481	 523	    1835008
      2097280	11814282322	 677	    2097152
      2097280	14380475033	 556	    2097152
      2359424	11732158553	 681	    2359296
      2359424	14310057139	 559	    2359296
      2621568	11151421728	 717	    2621440
      2621568	13667203078	 585	    2621440
      2883712	 9635555389	 830	    2883584
      2883712	11857582908	 674	    2883584
      3145856	 8830491065	 905	    3145728
      3145856	10975456216	 728	    3145728
      4194432	 5703033760	1402	    4194304
      4194432	 7462653658	1072	    4194304
      5243008	 3764860746	2124	    5242880
      5243008	 5239989138	1526	    5242880
      6291584	 2970821306	2692	    6291456
      6291584	 4381387103	1825	    6291456
      7340160	 2625465946	3047	    7340032
      7340160	 4033162958	1983	    7340032
      8388736	 2448731494	3266	    8388608
      8388736	 3847558417	2079	    8388608
      9437312	 2352617641	3400	    9437184
      9437312	 3743372332	2137	    9437184
     10485888	 2288649712	3495	   10485760
     10485888	 3666709518	2181	   10485760
     12583040	 2222953585	3598	   12582912
     12583040	 3581563981	2233	   12582912
     14680192	 2195411712	3643	   14680064
     14680192	 3543382742	2257	   14680064
     16777344	 2180601210	3668	   16777216
     16777344	 3522084255	2271	   16777216
     20971648	 2170401128	3685	   20971520
     20971648	 3507422082	2280	   20971520
     25165952	 2168353166	3689	   25165824
     25165952	 3503128913	2283	   25165824
     29360256	 2167406475	3691	   29360128
     29360256	 3502572094	2284	   29360128
     33554560	 2167236950	3691	   33554432
     33554560	 3501908751	2284	   33554432
     67108992	 2167069057	3691	   67108864
     67108992	 3502477870	2284	   67108864
    134217856	 2167397762	3691	  134217728
    134217856	 3502216012	2284	  134217728
    268435584	 2167170987	3691	  268435456
    268435584	 3502080053	2284	  268435456
    536871040	 2167207622	3691	  536870912
    536871040	 3502124096	2284	  536870912
   1073741952	 2167288244	3691	 1073741824
   1073741952	 3501543006	2284	 1073741824
   2147483776	 2167167763	3691	 2147483648
   2147483776	 3501652699	2284	 2147483648
   4294967424	 2167332378	3691	 4294967296
   4294967424	 3501536124	2284	 4294967296
   8589934720	 2167234766	3691	 8589934592
   8589934720	 3501106948	2284	 8589934592
  17179869312	 2166899399	3691	17179869184
  17179869312	 3500085763	2285	17179869184
  % END TABTABLE SELECT testsize AS x, FLOOR(bandwidth), FLOOR(rate * 1e12), ar...

  %% TABTABLE SELECT testsize AS x, FLOOR(bandwidth), FLOOR(rate * 1e12), areasize
  %% FROM test ORDER BY x, bandwidth LIMIT 20
   1152	20004857425	399	 1024
   1152	21256634665	376	 1024
   2176	20059162540	398	 2048
   2176	21206797610	377	 2048
   3200	20101648154	397	 3072
   3200	21256759869	376	 3072
   4224	20078735758	398	 4096
   4224	21256469275	376	 4096
   6272	20103335011	397	 6144
   6272	21254940419	376	 6144
   8320	19844289277	403	 8192
   8320	21033549756	380	 8192
  12416	19934071922	401	12288
  12416	21105216385	379	12288
  16512	19970353548	400	16384
  16512	21142325436	378	16384
  20608	19997700036	400	20480
  20608	21164943036	377	20480
  24704	20017094792	399	24576
  24704	21177968064	377	24576
  % END TABTABLE SELECT testsize AS x, FLOOR(bandwidth), FLOOR(rate * 1e12), ar...

indentation end
this is the end
